script.js

```diff
@@ -0,0 +1,333 @@
+        break;
+      case ('◄'):
+        event.preventDefault();
+        document.querySelector('TEXTAREA').focus();
+        document.querySelector('TEXTAREA').selectionStart = document.querySelector('TEXTAREA').selectionStart - 1;
+        document.querySelector('TEXTAREA').selectionEnd = document.querySelector('TEXTAREA').selectionEnd - 1;
+        break;
+      case ('►'):
+        event.preventDefault();
+        document.querySelector('TEXTAREA').focus();
+        document.querySelector('TEXTAREA').selectionStart = document.querySelector('TEXTAREA').selectionStart + 1;
+        document.querySelector('TEXTAREA').selectionEnd = document.querySelector('TEXTAREA').selectionEnd + 1;
+        break;
+      default:
+        clickSelectionHandler(document.querySelector('TEXTAREA'), document.getElementById(event.code).innerText);
+        // document.querySelector('TEXTAREA').value += document.getElementById(event.code).innerText;
+    }
+
+    document.getElementById(event.code).classList.add('active');
+    document.querySelector('TEXTAREA').focus();
+  });
+
+  document.addEventListener('keyup', (event) => {
+    document.querySelectorAll('DIV').forEach((item) => {
+      item.classList.remove('active');
+      if (item.classList.contains('caps') && isCapslock) item.classList.add('active');
+    });
+
+    if (event.code === 'ShiftLeft' || event.code === 'ShiftRight') {
+      if (document.querySelector('.caps').classList.contains('active')) {
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.upperCaseEn, document.querySelector('.keyboard-styles'));
+        document.querySelector('.caps').classList.add('active');
+      } else {
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.lowerCaseEn, document.querySelector('.keyboard-styles'));
+      }
+    }
+    document.querySelector('TEXTAREA').focus();
+  });
+};
+
+const switchLanguage = () => {
+  document.addEventListener('keydown', (event) => {
+    // console.log('meow');
+    if (event.ctrlKey && event.altKey) {
+      event.preventDefault();
+      if (localStorage.getItem('isEng') === 'true' && document.querySelector('.caps').classList.contains('active')) {
+        localStorage.setItem('isEng', 'false');
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.upperCaseRu, document.querySelector('.keyboard-styles'));
+      } else if (localStorage.getItem('isEng') === 'false' && document.querySelector('.caps').classList.contains('active')) {
+        localStorage.setItem('isEng', 'true');
+        document.querySelector('.keyboard-styles').innerHTML = '';
+        drawKeys(dataObject.upperCaseEn, document.querySelector('.keyboard-styles'));
+      } else if (localStorage.getItem('isEng') === 'true' && !document.querySelector('.caps').classList.contains('active')) {
```

Comment:
For `localStorage.getItem('isEng') === 'true'` you could create helper `isEng`
```js
const isEng = () => localStorage.getItem('isEng') === 'true';
```
For `document.querySelector('.caps').classList.contains('active')` you could create helper `isCaps`
```js
const isCaps = () => document.querySelector('.caps').classList.contains('active');
```

Reply:
done